docs/source/tutorials/metpy.md

```diff
@@ -0,0 +1,113 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.8
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Using `metpy` for WRF analysis
+
++++
+
+In this tutorial, we will show you how `xWRF` enables a seamless integration of WRF data with [`metpy`](https://unidata.github.io/MetPy/latest/). In the end, we will have a skewT plot at a lat-lon location in the simulation domain. Much of this tutorial was adapted from [`metpy`](https://unidata.github.io/MetPy/latest/tutorials/upperair_soundings.html).
+
++++
+
+## Loading the data
+
++++
+
+First of all, we load the data and use the simple `.xwrf.postprocess()` API.
+
+```{code-cell} ipython3
+import xwrf
+
+ds = xwrf.tutorial.open_dataset("wrfout").xwrf.postprocess()
+ds
+```
+
+## Sampling
+
++++
+
+Then, we sample the dataset at the desired location using `pyproj`. I selected the German Weather Service station at Mannheim, which is close to the middle of the simulation domain.
+
+```{code-cell} ipython3
+import numpy as np
+from pyproj import Transformer, CRS
+
+
+def sample_wrf_ds_at_latlon(ds, lat, long):
+    trf = Transformer.from_crs(CRS.from_epsg(4326), ds.wrf_projection.item(), always_xy=True)
+    x, y = ([var] if np.isscalar(var) else var for var in trf.transform(long, lat))
+    return ds.interp(x=x, y=y, x_stag=x, y_stag=y)
+```
+
+```{code-cell} ipython3
+ds = sample_wrf_ds_at_latlon(ds, 49.5091090731333, 8.553966628049963)
+ds
+```
+
+## Computation of desired quantities
+
++++
+
+Now we have to compute the quantities [`metpy`](https://unidata.github.io/MetPy/latest/) uses for the skewT. For that we have to first quantify the WRF data and then convert the data to the desired units.
+
+```{code-cell} ipython3
+import metpy
+import metpy.calc as mpcalc
+import pint_xarray
```

Comment:
`pint-xarray` is missing from the [docs environment](https://github.com/ncar-xdev/xwrf/blob/main/ci/environment-docs.yml). As a result, this notebook fails to run

Reply:
Good catch, @andersy005!